src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -33,6 +33,10 @@ class CallActor(call: Call, locallyQualifiedInputs: Map[String, WdlValue], backe
 
   override def receive = LoggingReceive {
     case CallActor.Start => handleStart()
+    case VassalCallActor.Finished(outputs) => context.parent ! WorkflowActor.CallCompleted(call, outputs)
```

Comment:
Good catch but other way around. `CallActor.Start` should be reversed, the standard we've been using is the received message is owned by the actor